common/conf.py

```diff
@@ -178,7 +179,7 @@ def get_previous_version(majornumber, minornumber=0):
     'presentations': (downloads_root + '/presentations/%s', ''),
     # Doc links
     'model_doc': (docs_root + '/latest/ome-model/%s', ''),
-    'devs_doc': (docs_root + '/contributing/%s', ''),
+    'devs_doc': (rtd_docs_root + '/contributing/%s', ''),
```

Comment:
Is there another `/contributing` namespace? I would have thought the content would be published at the root.

Otherwise a RFE after this PR is integrated might be to rename this alias as `contributing_doc`.

Reply:
thanks for spotting it